packages/concerto-core/test/composer/i18n.js

```diff
@@ -23,7 +23,7 @@ const Factory = require('./../../lib/factory');
 const Serializer = require('./../../lib/serializer');
 const ModelManager = require('./../../lib/modelmanager');
 const ModelUtil = require('./../../lib/modelutil');
-const Util = require('../composer/systemmodelutility');
+const Util = require('../composer/composermodelutility');
 const Moment = require('moment-mini');
 
 describe('Globalization', function() {
```

Comment:
Redundant?

Reply:
> Redundant?

Redundant with what?

Reply:
To clarify: `composermodelutility` is now a user model as any other. It's been used throughout all the tests though, so kept it for testing.

Reply:
Ah, I wrote this comment before I realised that this was just a utility file for tests